internal/template/funcmap.go

```diff
@@ -24,17 +24,18 @@ import (
 )
 
 var funcMap = map[string]interface{}{
-	"get":             get,
-	"has":             has,
-	"hcl":             hcl,
-	"hclField":        hclField,
-	"merge":           merge,
-	"replace":         replace,
-	"resourceName":    resourceName,
-	"now":             time.Now,
-	"trimSpace":       strings.TrimSpace,
-	"regexReplaceAll": regexReplaceAll,
-	"makeSlice":       makeSlice,
+	"get":               get,
+	"has":               has,
+	"hcl":               hcl,
+	"hclField":          hclField,
+	"merge":             merge,
+	"replace":           replace,
+	"resourceName":      resourceName,
+	"now":               time.Now,
+	"trimSpace":         strings.TrimSpace,
+	"regexReplaceAll":   regexReplaceAll,
+	"makeSlice":         makeSlice,
+	"schemaDescription": schemaDescription,
```

Comment:
where is this function used?

Reply:
Are you planning to use it in following PRs in recipe/component templates?

Reply:
I just decided to separate this minor update from the actual implementations. There are two PRs using this.

Look at [this example ](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/965/files#diff-302c27f4dbc9649902ac53af0fb0be28dc03d8bb5883d76a98a27743c8d71fddR19)

Reply:
Got it. Thanks.